docs/setup_vscode.md

```diff
@@ -480,189 +414,144 @@ To configure input redirection, edit `launch.json`.
     ]
 }
 ```
+{: data-title="launch.json" data-highlight="10" }
 
-### macOS M1 chip `launch.json` changes
+### Arguments and options
+<div class="primer-spec-callout info" markdown="1">
+Skip this subsection for EECS 280 project 1.
+</div>
 
-To configure input redirection, edit `launch.json`.
+Arguments and options are inputs to a program typed at the command line.  For example:
+```console
+$ ./main.exe train_small.csv test_small.csv --debug
+```
+{: data-variant="no-line-numbers" data-highlight="1" }
+
+- `main.exe` is the name of the program
+- `train_small.csv` and `test_small.csv` are arguments
+- `--debug` is an option
+
+To run a program with options or arguments in VS Code, edit `launch.json`.  Each option or argument should goes in a separate comma-separated string.
 ```json
 {
     "configurations": [
         {
             ...
-            "stdio": ["main_test.in", null]
+            "program": "${workspaceFolder}/src/main.exe",
+            "args": ["train_small.csv", "test_small.csv", "--debug"],
             ...
         }
     ]
 }
 ```
+{: data-title="launch.json" data-highlight="6" }
 
+## Debug
+Copy this example `main.cpp` into your editor.
+```c++
+#include <iostream>
+#include <vector>
+using namespace std;
 
-## Arguments and options
-<div class="primer-spec-callout info" markdown="1">
-Skip this subsection for EECS 280 project 1.  You'll need it for project 2 and beyond.
-</div>
-
-*Arguments* and *options* are inputs to a program typed at the command line.  Arguments are often required.  Options (AKA *flags* or *switches*) start with a hyphen (`-`), and are typically optional.
+double sum (vector<double> data) {
+  double total = 0;
+  for (auto i:data) {
+    total += i;
+  }
+  return total;
+}
 
-**Arguments example** from project 2:  `resize.exe` is the name of the program, and the arguments are `horses.ppm`,  `horses_400x250.ppm`, `400`, and `250`.
-```console
-$ ./resize.exe horses.ppm horses_400x250.ppm 400 250
+int main() {
+  vector<double> data;
+  data.push_back(10);
+  data.push_back(20);
+  data.push_back(30);
+  cout << "sum(data) = " << sum(data) << "\n";
+}
 ```
 
```

Comment:
Perhaps a snippet of the expected launch.json as well:
{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "name": "(lldb) Launch",
      "type": "cppdbg",
      "request": "launch",
      "program": "${workspaceFolder}/src/main.exe",
      "args": [],
      "stopAtEntry": false,
      "cwd": "${fileDirname}",
      "environment": [
        {
          "name": "ASAN_OPTIONS",
          "value": "abort_on_error=1:detect_leaks=0"
        }
      ],
      "externalConsole": false,
      "MIMode": "lldb",
    }
  ]
}

Students may still have the arguments and setUpCommands from the prior section.

Reply:
I'm hesitant to do this because students on different platforms will have different `launch.json` contents.